data/dose_entry.go

```diff
@@ -0,0 +1,97 @@
+package data
+
+import (
+	"time"
+
+	"github.com/tidepool-org/platform/structure"
+)
+
+const (
+	MinValue = 0
+	MaxValue = 1000
```

Comment:
```suggestion
	MaxValue = 100
```
Consistent with `BolusNormal`

Reply:
ok